tests/util/run_services.sh

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+
+set -euo
+
+INTEGRATION_LOG_PATH=/tmp/dashboard-integration-test.log
+TIUP_BIN_DIR=$HOME/.tiup/bin
+PLAYGROUND_TAG="integration_test"
+
+PROJECT_DIR="$(dirname "$0")/.."
+BIN="${PROJECT_DIR}/bin"
+
+start_tidb() {
+  echo "+ Waiting for TiDB start..."
+
+  rm -rf $INTEGRATION_LOG_PATH
+  TIDB_VERSION=${1:-latest}
+  $TIUP_BIN_DIR/tiup playground --tag $PLAYGROUND_TAG $TIDB_VERSION > $INTEGRATION_LOG_PATH &
+  ensure_tidb
+
+  echo "  - Start TiDB@$TIDB_VERSION Success!"
+}
+
+stop_tidb() {
+  echo "+ Waiting for TiDB teardown..."
+  $TIUP_BIN_DIR/tiup clean $PLAYGROUND_TAG >/dev/null 2>&1
+}
+
+ensure_tidb() {
+  i=1
+  while ! grep "CLUSTER START SUCCESSFULLY" $INTEGRATION_LOG_PATH; do
+    i=$((i+1))
+    if [ "$i" -gt 20 ]; then
+      echo 'Failed to start TiDB'
+      return 1
+    fi
+    sleep 5
+  done
+}
+
+FIXTRUE_DIR="${PROJECT_DIR}/tests/fixtures"
+
+import_test_data() {
+  if [ -e "$BIN/tidb-lightning" ]; then
+    echo "+ Start import fixtures..."
+    $BIN/tidb-lightning --backend tidb -tidb-host 127.0.0.1 -tidb-port 4000 -tidb-user root -d $FIXTRUE_DIR
+    echo "  - Import success!"
+  else
+    echo "Tool $BIN/tidb-lightning not exist"
+    return 1
+  fi
+}
+
+dump_test_data() {
+  if [ ${1:-""} = "" ]; then
+    echo "Please specify the 'database-name.table-name' to dump"
+    echo "Usage: tests/dump.sh database-name.table-name"
+    return 1
+  fi
+
+  if [ -e "$BIN/dumpling" ]; then
+    echo "+ Start dump fixtures..."
+    $BIN/dumpling -u root -P 4000 -h 127.0.0.1 --filetype sql -o $FIXTRUE_DIR -T $1
+    echo "  - Dump success!"
+  else
+    echo "Tool $BIN/dumpling not exist"
+    return 1
+  fi
```

Comment:
The official mysqldump is enough.

Reply:
mysqldump 8.0 looks like not supporting dump the INFORMATION_SCHEMA schema.
```
mysqldump does not dump the INFORMATION_SCHEMA schema.
```
https://dev.mysql.com/doc/refman/8.0/en/mysqldump.html